src/FinancialReports/HeaderExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Header Extension.
+ * This file goes in the twig/twig/src folder
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2019 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Twig;
+
+use Twig\Extension\AbstractExtension;
```

Comment:
hi @juggernautsei , while redundant namespace will now change to `namespace OpenEMR\FinancialReports;`

Reply:
so will this be where the file lives? It does not have to live in the twig src directory?

Reply:
right, it lives in your src directory but uses a Twig class